tests/integration/test_charm.py

```diff
@@ -391,3 +394,26 @@ async def test_audit_log(ops_test: OpsTest) -> None:
         item = json.loads(line)
         # basic sanity check
         assert audit_log_line_sanity_check(item), "Audit sanity log check failed for first line"
+
+
+@pytest.mark.group(1)
+async def test_log_roate(ops_test: OpsTest) -> None:
+    """Test that log are being rotated."""
+    app_name = await get_app_name(ops_test)
+    await start_continous_writes(ops_test, 1)
+    # Note: this timeout out depends on max log size
+    # which is defined in "src/config.py::Config.MAX_LOG_SIZE"
+    time.sleep(60 * 10)
+    await stop_continous_writes(ops_test, down_unit="mongodb/0", app_name=app_name)
+    # Just to make sure that logroate will run
+    time.sleep(60)
```

Comment:
a constant to replace `60` with `TIME_FOR_LOG_ROTATE` would be great

Reply:
Done